frontend/packages/console-app/src/components/cloud-vm-console/cloud-vm-console.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import ConnectedVMConsole from '@console/kubevirt-plugin/src/components/connected-vm-console/connected-vm-console';
```

Comment:
@glekner Let's leverage #5645 and move all VM-related console code to `kubevirt-plugin`.